staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4270,20 +4270,28 @@ const (
 	ServiceInternalTrafficPolicyLocal ServiceInternalTrafficPolicyType = "Local"
 )
 
-// ServiceExternalTrafficPolicyType describes how nodes distribute service traffic they
+// ServiceExternalTrafficPolicy describes how nodes distribute service traffic they
 // receive on one of the Service's "externally-facing" addresses (NodePorts, ExternalIPs,
-// and LoadBalancer IPs).
```

Comment:
is this on purpose? or you delete this line accidentally?

Reply:
nope, mistake